charts/minecraft/Chart.yaml

```
@@ -1,7 +1,6 @@
 apiVersion: v1
 name: minecraft
 version: 2.0.4
```

Bump the patch version, please:
```suggestion
version: 2.0.5
```

---

👍